walsh_ao5A.js

```diff
@@ -0,0 +1,43 @@
+/* Tyler Walsh
+walsh_ao5A.js
+19WI_INFO_2124_WW
+Thoendel
+1/18/2019 */
+
+const toDoList = []; //An array to keep track of my to-do-list. Created with an array literal.
+
+toDoList[0] = "Buy Groceries"; //The first item on my to-do-list.
+toDoList[1] = "Change Tire"; //The second item on my to-do-list.
+toDoList[2] = "Vacuum"; //The third item on my to-do-list.
+toDoList[3] = "Make Dinner"; //The fourth item on my to-do-list.
+// console.log(toDoList); Outputs the list to check what order things are in. Exactly as listed above.
+
+toDoList.unshift("Water Plants"); //Adds an item to the beginning of the list.
+// console.log(toDoList); Outputs the list to check what order things are in. As above, but with "water plants" at the beginning.
+
+
+toDoList.reverse(); //Reverses the order of the list.
+toDoList.sort(); //Sorts the list. Since this list uses strings and no other method of sorting was input, it sorts alphabetically.
+
+toDoList.includes("x"); //Checks if my to-do-list contains a lowercase 'x'.
```

Comment:
Good check! I'd love to see you do something with this value - like output it to the console so that we know if x is in the list.